<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWidget</class>
 <widget class="QWidget" name="StatisticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>from: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateFrom">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateTo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="twCalculation">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>220</height>
        </size>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Revenue:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Material Costs:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Labor Costs:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Sales Tax:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Pre Tax:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Profit:</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>35000€</string>
        </property>
        <property name="textAlignment">
         <set>AlignTrailing|AlignVCenter</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>10000€</string>
        </property>
        <property name="textAlignment">
         <set>AlignTrailing|AlignVCenter</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>1000€</string>
        </property>
        <property name="textAlignment">
         <set>AlignTrailing|AlignVCenter</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>2000€</string>
        </property>
        <property name="textAlignment">
         <set>AlignTrailing|AlignVCenter</set>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>1000€</string>
        </property>
        <property name="textAlignment">
         <set>AlignTrailing|AlignVCenter</set>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>24000€</string>
        </property>
        <property name="textAlignment">
         <set>AlignTrailing|AlignVCenter</set>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
